### DISCUSSION TITLE
How to customize the version or name of log file

### DISCUSSION THREAD

**Discussion by Les1ie** [Lightning-AI/pytorch-lightning#7897] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
`v_num` in progress bar means the version number of this running, and the log file's save directory is `version_{v_num}`.

How can i customize the directory's name, such as `version_GAT`, `version_GCN`.

**Comment by tshu-w** [comment, marked as answer]
you change it by passing your logger:
```python
from pytorch_lightning import Trainer
from pytorch_lightning.loggers import TensorBoardLogger
logger = TensorBoardLogger("tb_logs", name="my_model", version="GAT")
trainer = Trainer(logger=logger)
```

**Reply by Les1ie** [reply]
Thanks !!!!

**Reply by tshu-w** [reply]
@Les1ie Maybe you can select my comment as answer 😃
